templates/error/general_http_error.json.twig

```diff
@@ -0,0 +1,13 @@
+{
+    "error": {
+        {% block title %}
+        "location": "{{ pageTitle|attr_js }}",
+        {% endblock %}
+        {% block statusCode %}
+        "code": {{ statusCode|attr_js }},
+        {% endblock %}
+        {% block message %}
+        "message": "{{ errorMessage|default("Please try your request again, check the server logs, or contact the site administrator for support")|xlt|attr_js}}"
```

Comment:
just showed the lines with variables below
```
"location": {{ pageTitle|json_encode }},
"code": {{ statusCode|json_encode }},
"message": {{ errorMessage|default("Please try your request again, check the server logs, or contact the site administrator for support")|xl|json_encode}}
```

Reply:
fixed